### PR TITLE
New sample: http_client

### PR DESCRIPTION
**Pull request by weinrank**

### CODE REVIEW

File: programs/http_client.c [hunk 1]
```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (C) 2015 Felix Weinrank
```
Comment (tuexen) [inline programs/http_client.c]: Living in the past?

File: programs/http_client.c [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+ * OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ */
+
+/*
+ * Usage: http_client remote_addr remote_port [local_port] [local_encaps_port] [remote_encaps_port]
+ */
```
Comment (tuexen) [inline programs/http_client.c]: Would it make sense to add the object which is requested as a parameter?
That way we could request `cgi-bin/he` to get the transport protocol choice information...

File: programs/http_client.c [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+		}
+	}
+	while (!done) {
+#ifdef _WIN32
+		Sleep(1*1000);
+#else
+		sleep(1);
+#endif
+	}
+	usrsctp_get_stat(&stat);
```
Comment (tuexen) [inline programs/http_client.c]: Not sure this is useful here...

File: programs/http_client.c (hunk 2 above)
Comment (weinrank) [inline programs/http_client.c]: feature added in a147886

File: programs/http_client.c (hunk 1 above)
Comment (weinrank) [inline programs/http_client.c]: typo fixed in a147886

File: programs/http_client.c (hunk 3 above)
Comment (weinrank) [inline programs/http_client.c]: removed in a147886

**Pull request merged**